src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
> I had edited my comment to say I'll add a layer of deprecation.

cool, thanks

> The helpers add nothing on top of `<a-event>`.

it might be slight, but being able to visually scan and see the event name helps a ton with readability, not to mention a couple fewer characters. think of it like `<div>` versus `<span style="display: block">`.

> They can actually limit developers to think they can only handle a limited number of events, when actually they are able to handle any event. 

> Plus, some of them are misleading because they don't map to the actual event name.

just `<a-hover>`; you can remove that one if you want w/o deprecation - haven't seen anyone use that one (but if you want to be safe keep it).